bundles/AdminBundle/Controller/Admin/RedirectsController.php

```diff
@@ -264,4 +264,28 @@ public function cleanupAction()
             return $this->adminJson(['success' => false]);
         }
     }
+
+    /**
+     * @Route("/get-statuscodes", name="pimcore_admin_redirects_statuscodes", methods={"GET"})
+     *
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    public function statusCodesAction()
+    {
+        $this->checkPermission('redirects');
+        $statusCodes = Redirect::getStatusCodes();
+        $codes = [];
+        foreach ($statusCodes as $statusCode => $label) {
+            $codes[] = [
+                'statusCode' => $statusCode,
+                'display' => $label . " ($statusCode)",
```

Comment:
I think mixing variable expansion and string concatenation is somehow strange and unnecessary:
```suggestion
                'display' => "$label ($statusCode)",
```